template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatahubConfigProviderTest.java

```diff
@@ -42,7 +43,8 @@ static Object[][] razCloudPlatformDataProvider() {
         return new Object[][]{
                 // testCaseName cloudPlatform
                 {"CloudPlatform.AWS", CloudPlatform.AWS},
-                {"CloudPlatform.AZURE", CloudPlatform.AZURE}
+                {"CloudPlatform.AZURE", CloudPlatform.AZURE},
+
```

Comment:
GCP is missing?

Reply:
GCP is not added here since raz is not applicable for datahub in GCP. This dataset is being used for AWS and Azue to check if config is added or not by varying the cm version. For GCP, I have added another test getAdditionalServicesWhenRazIsEnabledForGcpWithCm790 to check that config is not added even if CM version is supported since Raz on Datahub is not supported.